libraries/features/dma/simulation/dramsim3.mk

```diff
@@ -64,6 +64,7 @@ $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -I$(BASEJU
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -I$(BASEJUMP_STL_DIR)/imports/DRAMSim3/ext/fmt/include
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DFMT_HEADER_ONLY=1
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DBASEJUMP_STL_DIR="$(BASEJUMP_STL_DIR)"
+$(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DBLOOD_GRAPH=1
```

Comment:
Always on?

Reply:
Hrm. Probably not? We could make a version of libdramsim3 with blood graphs compiled in and then only link when profiling. That would more or less match what we do for the vanilla blood graphs right? With the tracer?

Reply:
Do DRAMSim3 bloodgraphs slow down sim?

Reply:
I'm going to go ahead and say "probably"

Reply:
Then we should come up with a way of turning it on and off, unless we know otherwise

Reply:
Maybe I can tackle that in a separate PR. It's not super integral to this one.